<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateCategoryWindow</class>
 <widget class="QWidget" name="UpdateCategoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="categoryNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Category Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryNameInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
